batcher/aligned-batcher/src/connection.rs

```diff
@@ -40,12 +43,15 @@ pub(crate) async fn send_batch_inclusion_data_responses(
             .await;
 
         match sending_result {
-            Err(Error::AlreadyClosed) => (),
-            Err(e) => error!("Error while sending batch inclusion data response: {}", e),
-            Ok(_) => (),
+            Err(Error::ConnectionClosed) | Err(Error::AlreadyClosed) | Err(Error::Io(_)) => {
+                metrics::BROKEN_SOCKETS_LATEST_BATCH.inc();
+                error!("Error while sending the batch response, socket connection was closed!");
+            }
+            Err(e) => {
```

Comment:
I believe we should also take into account these cases in the metrics, but I am not sure.